### PR TITLE
MOD-5706: Exclude and deny-script for internal cluster commands

### PR DESCRIPTION
**Pull request by oshadmi**
**Main objects this PR modified**
* Exclude internal commands
  * Do not exclude commands needed by DMC
* Deny running them using script

**Mark if applicable**

- [ ] This PR introduces API changes
- [ ] This PR introduces serialization changes

[MOD-5706](https://redislabs.atlassian.net/browse/MOD-5706)

[MOD-5706]: https://redislabs.atlassian.net/browse/MOD-5706?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: coord/src/module.c [hunk 1]
```diff
@@ -2544,8 +2544,8 @@ RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 
   // cluster set commands
   RM_TRY(RedisModule_CreateCommand(ctx, REDISEARCH_MODULE_NAME".CLUSTERSET", SafeCmd(SetClusterCommand), "readonly allow-loading", 0,0, -1));
```
Comment (MeirShpilraien) [inline coord/src/module.c:2546]: Lets also add the deny-script to `CLUSTERSET`.

File: pack/ramp-light.yml [hunk 2]
```diff
@@ -28,3 +28,5 @@ capabilities:
 exclude_commands:
     - FT.CONFIG
     - FT.DEBUG
+    - search.CLUSTERREFRESH
+    - search.CLUSTERINFO
```
Comment (MeirShpilraien) [inline pack/ramp-light.yml:32]: Is this really needed? Those commands are not even expose on search lite.

**Review by MeirShpilraien — COMMENTED** [review]
Lets add a test that verify that those 3 commands can not be called from within a script.

File: tests/pytests/test.py [hunk 3]
```diff
@@ -3763,3 +3763,24 @@ def verify_address(addr):
                'MASTER'
             ).equal('OK')
     verify_address('::1')
+
+def test_internal_commands(env):
+    ''' Test that internal cluster commands cannot run from a script '''
+    if not env.is_cluster():
+        env.skip()
+    env.cmd('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT')
```
Comment (MeirShpilraien) [inline tests/pytests/test.py:3771]: Why do you need to create an index?

File: pack/ramp-light.yml (hunk 2 above)
Comment (MeirShpilraien) [reply pack/ramp-light.yml:32]: @oshadmi what about this?

**Review by MeirShpilraien — COMMENTED** [review]
Some more comments

File: tests/pytests/test.py [hunk 4]
```diff
@@ -3763,3 +3763,24 @@ def verify_address(addr):
...
+
+    def fail_eval_call(r, env, cmd):
+        cmd = str(cmd)[1:-1]
```
[the remark this answers is not in the document]
Comment (oshadmi) [reply tests/pytests/test.py:3773]: Unpack a list of strings to a comma separated list of strings

[the remark this answers is not in the document]
Comment (MeirShpilraien) [reply tests/pytests/test.py:3775]: Ohh I see, OK, maybe we should add a comment explain this because its not trivial.

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**